std_msgs/String cellid
std_msgs/String essid
std_msgs/String mac_address
std_msgs/String signal_strength
std_msgs/String age
std_msgs/String channel
std_msgs/String signal_to_noise_ratio